### PR TITLE
Added CassandraGenerator import to oddrn_generator module

### PR DESCRIPTION
**Pull request by andrwqa**
Added CassandraGenerator import to oddrn_generator module.

### CODE REVIEW

File: oddrn_generator/path_models.py [hunk 1]
```diff
@@ -134,6 +135,7 @@ class SnowflakePathsModel(BasePathsModel):
 
     class Config:
         dependencies_map = {
+            "warehouses": ("warehouses",),
```
Comment (Vixtir) [inline oddrn_generator/path_models.py:138]: Why did you add that?

Comment (andrwqa) [reply oddrn_generator/path_models.py:138]: otherwise pytests fail
